backend/common/db/models/simulation/__init__.py

```
@@ -1,4 +1,18 @@
-__all__ = []
+__all__ = [
+    "UserProgress",
+    "SceneProgress",
+    "ConversationLog",
+    "ConversationSummaries",
+    "AgentSessions",
+    "SessionMemory",
+    "VectorEmbeddings",
+    "GradingMaterial",
+    "GradingMaterialChunk",
+]
```

Ruff (0.14.8)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)